# Service for trading landmarks
string name
geometry_msgs/Pose2D pose
multi_agent_coverage_planner/Point2DArray landmarks
------
bool success
int32[] indexes_to_remove
multi_agent_coverage_planner/Point2DArray landmarks_to_add
